tests/unit/helpers/eventsRepetition.test.js

```diff
@@ -416,23 +475,27 @@ describe('createRepetitionsEveryWeekDay', () => {
   });
 });
 
-describe('createRepetitionsByWeek', () => {
+describe('createRepetitionsByWeek #tag', () => {
   let createRepeatedEvents;
+  let dayDiff;
   beforeEach(() => {
     createRepeatedEvents = sinon.stub(EventsRepetitionHelper, 'createRepeatedEvents');
+    dayDiff = sinon.stub(datesHelper, 'dayDiff');
   });
   afterEach(() => {
     createRepeatedEvents.restore();
+    dayDiff.restore();
   });
 
-  it('should create repetition by week', async () => {
+  it('should create repetition by week from first event to 90 days after', async () => {
```

Comment:
should create repetition by week with first event in the future

Reply:
done